<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="home-laptop" failures="1" tests="11" name="Test" time="17.793" errors="0" timestamp="2023-05-27T17:42:30 IST">
  <testcase classname="com.notesapi.tests.UsersTests" name="createNewUser" time="7.182"/>
  <testcase classname="com.notesapi.tests.UsersTests" name="performLogin" time="2.183">
    <failure type="java.lang.AssertionError" message="expected [201] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [201] but found [200]
at com.notesapi.tests.UsersTests.performLogin(UsersTests.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- performLogin -->
  <testcase classname="com.notesapi.tests.UsersTests" name="retrieveUserInfo" time="1.825"/>
  <testcase classname="com.notesapi.tests.UsersTests" name="editUserInfo" time="0.704"/>
  <testcase classname="com.notesapi.tests.UsersTests" name="editUserPassword" time="0.918"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="createNewNote" time="0.844"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="retrieveNote" time="0.593"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="retrieveNoteById" time="0.669"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="updateExistingNote" time="0.722"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="updateCompleteStatusOfNote" time="0.667"/>
  <testcase classname="com.notesapi.tests.NotesTests" name="deleteNoteById" time="0.694"/>
</testsuite> <!-- Test -->
